.gitignore

```diff
@@ -6,3 +6,6 @@ GalacticScale.sln.DotSettings.user
 DSP_Plugins.GalacticScale/Directory.Build.props
 DSP_Plugins.GalacticScale/lib/**
 GalacticScale2/Directory.Build.props
+GalacticScale2/obj/
```

Comment:
I would recommend just replacing the gitignore with the VisualStudio gitignore preset https://github.com/github/gitignore/blob/main/VisualStudio.gitignore